libbeat/_meta/config/output-elasticsearch.reference.yml.tmpl

```diff
@@ -77,6 +77,10 @@ output.elasticsearch:
   # Configure HTTP request timeout before failing a request to Elasticsearch.
   #timeout: 90
 
+  # {{.BeatName}} expects Elasticseach to be the same version or newer than the Beat.
```

Comment:
```suggestion
  # {{.BeatName}} expects Elasticsearch to be the same version or newer than the Beat.
```

Reply:
Thanks, the templated documentation files still look like they have the typo in them. Let's not forget to update those :)